chanzuckerberg/idseq-web
# [Mapping] Update the hover tooltip and add a basic popup window on click

### Description
- Update the hover tooltip to have the location name / sample count
- Add a basic onClick popup that will enumerate the samples at that point
- Refactor some of the demo locationsToSamples parsing
- Tweak the bubble size scaling
- Need to sync up with design on more details/adjustments
- Open to suggestions on what to name "hover tooltips" vs "onclick persistent popups" >:)

### Demo
![Screen Shot 2019-04-19 at 3 51 00 PM](https://user-images.githubusercontent.com/5652739/56447154-87e74d80-62bb-11e9-85d1-b78d995172fd.png)
![Screen Shot 2019-04-19 at 3 51 16 PM](https://user-images.githubusercontent.com/5652739/56447156-89b11100-62bb-11e9-9a82-ef604f34e2c7.png)

### Test and Reproduce
- Pull the branch, retrieve MAPTILER_API_KEY from Parameter Store, start locally with `MAPTILER_API_KEY=key_here docker-compose up web`
- Visit http://localhost:3000/locations/map_playground
- Hover over a bubble to see the hover. Click on a bubble to see the onClick popup.
